src/Logger/LoggerInterface.php

```diff
@@ -10,4 +10,11 @@ interface LoggerInterface
      * try to log this information.
      */
     public function logAssignment(AssignmentEvent $assignmentEvent): void;
+
+    /**
+     * Method used by EppoClient to log bandit action selection to the data warehouse.
+     *
+     * Logging Bandit selection events is crucial to the data pipeline driving decisions.
+     */
+    public function logBanditAction(BanditActionEvent $banditActionEvent): void;
```

Comment:
Are we forcing every client to implement this then (even if just a no-op)?

Reply:
no anymore ;)